packages/react-sdk/src/db/style-sources.ts

```diff
@@ -0,0 +1,34 @@
+import { z } from "zod";
```

Comment:
Here is what I would call them:

StyleSourcesToken > StyleSourceToken
StyleSourcesLocal > StyleSourceLocal

StyleSourcesItem > StyleSource
StyleRefsItem > StyleSourceSelection

StyleRefs > StyleSourceSelections

Reply:
lets create project-build package